lib/request_handler/concerns/config_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module RequestHandler
+  module Concerns
+    module ConfigHelper
```

Comment:
As a slightly different approach, the config values could be memoized here or in the respective Parser class via a hash map instead? This lookup is called on every request (potentially multiple times given the params), although there is probably only a small number of available lookup keys. If you just build a map
`(lookup_key => configured_value)`
then it wouldn't have to split the key string + cast every element on every request.

Reply:
Never mind, the levels of duration we're talking about are completely negligible